oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -307,10 +307,22 @@ private boolean indexProperty(String path,
     /**
      * Returns {@code true} if nodeScopeIndex full text values need to be indexed
      */
+    @Deprecated
     protected boolean isFulltextValuePersisted() {
         return true;
     }
 
+    /**
+     * In elastic we don't add analyzed data in :fulltext if index has both analyzed
+     * and nodescope property. Instead we fire a multiMatch with cross_fields.
+     *
+     * Returns {@code true} if nodeScopeIndex full text values need to be indexed
```

Comment:
Hm, I'm a bit confused... For me, the description of the return value doesn't fully match the function name? If we write "needs to be indexed", then the name of the function should probably be "isFulltextValueToBeIndexed"? Or if the the function name is correct, then the javadoc should be "needs to be persisted"... Not sure which one is the right answer; I could also be that both are correct and I just don't understand it yet :-)

Reply:
Changed to isFulltextValuePersistedAtNode